Framework/Core/src/WorkflowHelpers.cxx

```diff
@@ -44,6 +44,9 @@
 namespace o2::framework
 {
 
+static constexpr std::array<header::DataOrigin, 3> AODOrigins{header::DataOrigin{"AOD"}, header::DataOrigin{"A0D"}, header::DataOrigin{"A1D"}};
+static constexpr std::array<header::DataOrigin, 5> extendedAODOrigins{header::DataOrigin{"AOD"}, header::DataOrigin{"A0D"}, header::DataOrigin{"A1D"}, header::DataOrigin{"DYN"}, header::DataOrigin{"AMD"}};
```

Comment:
According to Giulio we can use the full 4 characters. So the additional two could be
AOD1 AOD2
A0D with the number instead of the O may be misleading...

Reply:
While header::Origin is indeed 4-byte, the string literals in C have an extra `\0` symbol at the end. So unless we explicitly set the bytes one-by-one, we cannot use 4-letter origins. 

I will add the error (and the test), but I wanted to also start the discussion on the origin names. Would A1D and A2D be better?

Reply:
@ktf told me it does not have to be zero-terminated
so in practice the current one is
AOD\0
So the new ones could also become
AOD\1
AOD\2
(if we make a define for them)
@ktf can you confirm?

Reply:
By original design, they are intended not to be 0 completed.

Reply:
I would not use \X though. Why not simply "AOD0", "AOD1", etc?